SConstruct

```diff
@@ -56,8 +56,10 @@ Additional help command options:
 # SConstruct is parsed by Python 2. This seems to be the most robust
 # and simplest option that will reliably trigger an error in Python 2
 # and provide actionable feedback for users.
+python_min_build_support = "3.7"
 f"""
-Cantera must be built using Python 3.6 or higher. You can invoke SCons by executing
+Cantera must be built using Python {python_min_build_support} or higher.
+You can invoke SCons by executing
```

Comment:
The problem with changing this to make the version a variable is that the version won't be printed in the error message returned to the user. For example, if this is invoked using Python 2.7, the error is:
```
scons: Reading SConscript files ...
  File "/src/cantera/SConstruct", line 65

    f"""
Cantera must be built using Python {python_min_build_support} or higher.
You can invoke SCons by executing
    python3 `which scons`
followed by any desired options.
"""
```

Reply:
:+1: Realized this right after pushing but had to leave my computer. Fixed!